manifests/config.pp

```diff
@@ -9,6 +9,13 @@
           unless  => "test /etc/alternatives/java -ef '${java::use_java_alternative_path}'",
         }
       }
+      if $java::use_java_home != undef {
+        file_line { 'java-home-environment':
+          path  => '/etc/environment',
+          line  => "JAVA_HOME=${$java::use_java_home}",
+          match => 'JAVA_HOME=',
```

Comment:
In the file_line documentation, it looks like 'replace' defaults to true, which would mean if a match is found the current implementation would be to replace it with the updated java home for the environment.